app/theme/button.ts

```diff
@@ -1,11 +1,16 @@
 export const Button = {
   // Override the default dark mode theme
   variants: {
-    solid: ({ colorMode }: { colorMode: 'dark' | 'light' }) => ({
-      bg: colorMode === 'dark' ? 'primary.500' : 'primary.500'
-    }),
-    outline: ({ colorMode }: { colorMode: 'dark' | 'light' }) => ({
-      color: colorMode === 'dark' ? 'primary.700' : 'primary.700'
```

Comment:
I deleted because it's more complicated to set up the original. We can just use ChakraUI default colours! And it's good for accessibility